asteroid/models/base_models.py

```diff
@@ -5,8 +5,17 @@
 import numpy as np
 
 from .. import torch_utils
-from ..utils.hub_utils import cached_download
 from ..masknn import activations
+from ..utils.hub_utils import cached_download
```

Comment:
Can we have this whole file in a separate PR please? 
This is quite important and I'd like to be able to dissociate from the rest. 
Thanks